pkgs/os-specific/linux/kernel/manual-config.nix

```
@@ -146,6 +142,11 @@ let
         fi
 
         patchShebangs scripts
+
+        # also patch arch-specific install scripts
+        for i in $(find arch -name install.sh); do
+            patchShebangs "$i"
+        done
```

```suggestion
        find arch -name install.sh -exec patchShebangs
```
also we only use 4 spaces in script only files.

---

Won't work, `patchShebangs` is a shell function.

---

> also we only use 4 spaces in script only files.

you missed the second point to this

---

Fixed that.